.github/workflows/telemetry/verify_micro_content.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+import requests
+import unittest
+
+
+class TestTelemetry(unittest.TestCase):
+    all = requests.request("GET", "http://127.0.0.1:9191/micro/all").json()
+    # {
+    #   "total": 55,
+    #   "good": 55,
+    #   "bad": 0
+    # }
+    good = requests.request("GET", "http://127.0.0.1:9191/micro/good").json()
+    #  [ ...{
+    # "unstruct_event": {
+    #     "schema": "iglu:com.snowplowanalytics.snowplow/unstruct_event/jsonschema/1-0-0",
+    #     "data": {
+    #         "schema": "iglu:com.snowplowanalytics.snowplow.telemetry/heartbeat/jsonschema/1-0-0",
+    #         "data": {
+    #             "userProvidedId": "userProvidedIdValue",
+    #             "moduleName": "moduleNameValue",
+    #             "moduleVersion": null,
+    #             "instanceId": null,
+    #             "region": null,
+    #             "cloud": null,
+    #             "applicationName": "snowplow-stream-collector",
```

Comment:
Ideally `applicationName` should be consistent with the terraform module, [which sets it to "stream-collector"](https://github.com/snowplow-devops/terraform-aws-collector-kinesis-ec2/blob/v0.2.0/main.tf#L5).

I actually think "stream-collector" is not a good name.  Because it is missing the `-kinesis` / `-sqs` / `-kafka` suffix.  (And less importantly, I never liked the "stream-" prefix, but that is more personal opinion).

I suggest you talk to Emily about what is a helpful application name for her.  I guess she will tell you it is helpful for her if terraform and collector send the same application name.